src/Microsoft.DotNet.Web.ProjectTemplates/content/StarterWeb-CSharp/Data/SqlLite/ApplicationDbContextModelSnapshot.cs

```diff
@@ -12,20 +12,66 @@
 namespace Company.WebApplication1.Data.Migrations
 {
     [DbContext(typeof(ApplicationDbContext))]
-    [Migration("00000000000000_CreateIdentitySchema")]
-    partial class CreateIdentitySchema
+    partial class ApplicationDbContextModelSnapshot : ModelSnapshot
     {
-        protected override void BuildTargetModel(ModelBuilder modelBuilder)
+        protected override void BuildModel(ModelBuilder modelBuilder)
         {
 #pragma warning disable 612, 618
             modelBuilder
 #if (UseLocalDB)
-                .HasAnnotation("ProductVersion", "2.1.0-preview1")
+                .HasAnnotation("ProductVersion", "2.1.0-preview1-28193")
```

Comment:
@ajcvickers / @bricelam - just checking, the version here is OK? I know we've discussed this a billion times but I can never remember, sorry!

Reply:
@Eilon It doesn't need to match anything so this is fine for preview, but we should update it before RTM ships--I filed https://github.com/aspnet/templating/issues/353 for this and assigned it to @HaoK